javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
Hi, I updated the PR the way we discussed it, but I'm still not sure if we can't update the previous records in the jv_snapshots with the corresponding versions. What is the field that is the same for all snapshots of a given entity (global_id_fk)? I'll try some combination of join/update/count query :)